playwright/src/main/java/com/microsoft/playwright/assertions/SoftAssertionsImpl.java

```diff
@@ -0,0 +1,42 @@
+package com.microsoft.playwright.assertions;
+
+import com.microsoft.playwright.Page;
+import org.opentest4j.AssertionFailedError;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class SoftAssertionsImpl implements SoftAssertions {
+  final List<Throwable> results;
```

Comment:
I would use `RuntimeException` here and in PageAssertionsImplProxy and SoftAssertionsBase. We don't want to catch any `Errors` anyway and only catch unchecked exceptions in SoftAssertionsBase anyway.

Reply:
@yury-s, we catch AssertionFailedError which is an Error

Reply:
Wow, didn't know that. Then `Throwable` is justified.